doc/contributing/building-node-with-ninja.md

```diff
@@ -32,6 +32,12 @@ make -j4 # With this flag, Ninja will limit itself to 4 parallel jobs,
          # regardless of the number of cores on the current machine.
 ```
 
+If you wish to only modify JS layer in `lib`, you can use:
+
+```bash
+./configure --ninja --node-builtin-modules-path "$(pwd)"
+```
+
```

Comment:
I would move this so it appears before the `make -j4` command – ideally just after the paragraph that mention `./configure`

Reply:
thanks for the suggestion, I have moved this to the `./configure` section.